py-rse/ci.Rmd

```diff
@@ -54,51 +51,40 @@ every time a pull request is merged.
 > it creates a new virtual machine to ensure that builds and tests run in a completely clean environment:
 > nothing left over from previous attempts to compile, run, or publish will affect the current attempt.
 
-## How do I set up continuous integration? {#py-rse-ci-basic}
+## Setting up continuous integration {#py-rse-ci-basic}
 
-We have to do five things to set up continuous integration:
+### Creating a Travis account to watch a repository
 
-1.  Create an account with [Travis CI][travis-ci].
-1.  Link our Travis CI account to our GitHub account.
-1.  Tell Travis CI to watch the repository that contains our project.
-1.  Create a `.travis.yml` file in your repository
-    to tell Travis CI what language we are working in
-    and what we want to do.
-1.  Commit that `.travis.yml` file to the repository.
+To set up CI for your project, you first must:
 
-> **There's More Than One Way to Do It**
->
-> This process is for general Travis CI usage.
-> If we are working with R projects or packages,
-> we can simplify things considerably using `usethis::use_travis`
-> (Section \@ref(py-rse-ci-setting-up-with-r)).
-
-Creating an account with an online service is a familiar process for most of us,
-but linking our Travis CI account to our GitHub account may be new.
-When we do this,
-we are not only telling Travis CI who we are on GitHub:
-we are also telling GitHub that it's OK for Travis CI to get information about us
-via GitHub's [API][api].
-This permission is necessary—without it,
-we would have to enter our password every time Travis CI wanted to run,
-which would hardly count as "automated"—but
-we should always check carefully when authorizing a service like this
-to make sure we are comfortable with what it is planning to do.
+1.  Create an account on [Travis CI][travis-ci].
+1.  Link your Travis CI account to your GitHub account.
+1.  Tell Travis CI to watch the repository that contains your project.
 
-### Watching a repository
+Creating an account with an online service may likely be a familiar process,
+but linking your Travis CI account to your GitHub account may be something new.
+Linking Travis to GitHub allows Travis to access all your GitHub repositories
+via GitHub's [API][api] in order to build and run the project.
```

Comment:
do we need to say that its via the GitHub API? i.e., does mention of the mechanics add value?